examples/esp32-with-cred/esp32-server-with-cred.rs

```diff
@@ -1,10 +1,10 @@
 use log::*;
 use thiserror::Error;
 
-use esp_idf_svc::eventloop::EspSystemEventLoop;
-use esp_idf_svc::nvs::{EspDefaultNvs, EspDefaultNvsPartition, EspNvs};
-use esp_idf_sys::EspError;
-use esp_idf_sys::{g_wifi_feature_caps, CONFIG_FEATURE_CACHE_TX_BUF_BIT};
+use micro_rdk::esp_idf_svc::eventloop::EspSystemEventLoop;
```

Comment:
Within examples, which don't really form part of micro-rdk, we obtain `esp-idf-svc` via the `micro-rdk` re-export.